pkg/identityserver/store/oauth_store.go

```diff
@@ -46,6 +46,7 @@ func (s *oauthStore) ListAuthorizations(ctx context.Context, userIDs *ttnpb.User
 		countTotal(ctx, query.Model(UserSession{}))
 		query = query.Limit(limit).Offset(offset)
 	}
+	setTotal(ctx, uint64(len(authModels)))
```

Comment:
I'm pretty sure this won't work, since `authModels` is populated on the line below this one.